src/docbuilder/rustwide_builder.rs

```diff
@@ -225,7 +226,7 @@ impl RustwideBuilder {
                     let file_name = if versioned {
                         format!("{}-{}.{}", segments[1], rustc_version, segments[0])
                     } else {
-                        file.to_string()
+                        (*file).to_string() // &str has a specialized ToString impl, while &&str goes through Display
```

Comment:
I don't think this dereference is necessary, `file` is already a `&str` because of the implicit dereference in the pattern on line 223.